### DISCUSSION TITLE
Use a 1.69 IPS and Touch screen for my nano Edgetx

### DISCUSSION THREAD

**Discussion by NSG-666** [EdgeTX/edgetx#5483] [Show and tell]
Use a 1.69 IPS and Touch screen for my nano Edgetx, and i make a video for it:
https://youtu.be/Y0F-h7VCCpE

https://github.com/user-attachments/assets/df724895-4d7b-48f6-8773-be178d8a5390

**Comment by pfeerick** [comment]
Since I see an Espressif can... is this running the ESP32 fork of EdgeTX? Nicely done! I take it you have some dedicated recording and sound playback stuff happening also? You know you just *have* to get this running on a wristwatch sized/wearable package now don't you, since it looks like you're using a watch screen for that! :rofl: Don't know how you do the input controls though... some sort of bluetooth or wireless controllers you can hold in your hand? Maybe wear a "smart ring" on each hand so you can track orientation/position of each hand? :thinking: :laughing:

**Reply by NSG-666** [reply]
@pfeerick I want a nano Egtx for my little Flex banlance robot, the first i will use the espnow protocol for wireless link, next step want to add wifi video stream page in Egtx, next next is speech control，balabala, and so much idea to be done......

**Reply by pfeerick** [reply]
Nice... yeah, of you have ESPs on both ends, ESPNOW would be perfect, since it is just there. Please keep us updated on your progress... looks like a fun project :)

**Reply by NSG-666** [reply]
A little update for the Flex robot and Edgetx nano:
https://youtu.be/43i2E8TLNZg
